elide-core/src/main/java/com/yahoo/elide/core/RequestScope.java

```diff
@@ -398,4 +414,25 @@ public void saveObjects() {
     public boolean useFilterExpressions() {
         return useFilterExpressions;
     }
+
+    /**
+     * Print the permission check statistics
+     * @return the permission check statistics
+     */
+    public String printCheckStats() {
+        StringBuilder sb = new StringBuilder("Permission Check Statistics:\n");
+
+        Map<String, Long> result = new LinkedHashMap<>();
+        checkStats.entrySet().stream()
+                .sorted(Map.Entry.comparingByValue())
+                .forEachOrdered(e -> result.put(e.getKey(), e.getValue()));
```

Comment:
Why don't you append to the string builder here instead of putting them in the map - and then iterating over that again?